Doc/whatsnew/3.14.rst

```diff
@@ -514,6 +514,14 @@ sys
   from other interpreters than the one it's called in.
 
 
+tkinter
+-------
+
+* The methods :meth:`!tkinter.Misc.after` and :meth:`!tkinter.Misc.after_idle`
+  now support passing keyword arguments to the function.
```

Comment:
Misc is effectively a mixin class that is never instantiated and whose existence is an implementation detail.  People always call these methods on widgets.

```suggestion
* Make tkinter widget methods ``after`` and ``after_idle`` accept arguments passed by keyword.
```
There are other possible wordings, but as a tkinter user, this seems clear enough to me.

Reply:
I would use formatting ``` :meth:`!after` ```.